src/Explorer/Notebook/NotebookContentClient.ts

```diff
@@ -189,7 +192,7 @@ export class NotebookContentClient {
         const dir = xhr.response;
         const item = NotebookUtil.createNotebookContentItem(dir.name, dir.path, dir.type);
         item.parent = parent;
-        parent.children.push(item);
+        parent.children!.push(item);
```

Comment:
In general we should avoid `!` as it may lead to a runtime error. Instead use `?` here. I think that will give you the same result but safer

Reply:
I will change it after do research